source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -525,6 +525,14 @@ def __init__(self, output_type_serializer):
             help="choose the criteria used to compute pt journeys, feature in beta ",
         )
 
+        parser_get.add_argument(
+            "_jo",
+            type=BooleanType(),
+            hidden=True,
+            default=False,
+            help="do not delete journeys tagged for olympics",
```

Comment:
IMO: :fr: naming should be avoided and the olympics is also too specific.
```suggestion
            "_protect",
            type=BooleanType(),
            hidden=True,
            default=False,
            help="Enable protection (from deletion) for journeys protected because of subjective extra criteria",
```

Reply:
Well, the `olympics` tag will be included in the navitia response, and is named as such to be understandable by the client/frontend. 
I think being specific here is needed, and replacing the `olympics` tag  by `protected` will only make things more obscure.

On the name of the parameter, maybe replace `_jo` by `_olympics` ? It may be useful for testing other features related to olympics. 

poke @wassimbenaissa what do you think ?

Reply:
I'd agree with @pbench at this point, it's already specific, let's not make something generic, since we don't even know what the future holds (planning a feature for an unknown future often ends up being a feature for only one specific case).

But I do agree :gb: `_olympics` would be better than :fr: `_jo`.

Reply:
OK, you've understood my point about being generic (which also forces us to be able to explain what's done), the final decision is not a blocker either way for me.

Another underlying point (not blocking either IMO, especially as it is a hidden param) is, do we want:
1. `_olympics` > "large" parameter that is only protecting some journeys now but is probably going to do more (in that case maybe document it a bit differently?).
2. `_keep_olympics` > narrower parameter that describes better just what's done (protect journeys tagged "olympics").

Reply:
ok :+1: , let's go with the more specific one '_keep_olympics_journeys'